[IOS] Undefined symbols for architecture x86_64

I'm trying to run following run on ios sim (also tried on an ipad before had a similar error)
```rust
use skia_safe::{ Surface, SurfaceProps, PixelGeometry, ColorType, Paint, Rect, SurfacePropsFlags};
use skia_safe::gpu::gl;
use skia_safe::paint::Style;
use std::os::raw::{c_int, c_uint};
use skia_safe::gpu::{Context, BackendRenderTarget, SurfaceOrigin};

#[no_mangle]
pub extern fn native_init(width: *const c_int, height: *const c_int, buffer: *const c_uint) {
    let interface = gl::Interface::new_native().unwrap();
    let mut context = Context::new_gl(Some(&interface));
    let frame_buffer = gl::FramebufferInfo::from_fboid(buffer as u32);
    let target = BackendRenderTarget::new_gl((width as i32, height as i32), Some(0), 0, frame_buffer);
    let surface_props = SurfaceProps::new(SurfacePropsFlags::default(), PixelGeometry::Unknown);
    let surface_holder = Surface::from_backend_render_target(&mut context.unwrap(), &target, SurfaceOrigin::TopLeft, ColorType::RGBA8888, None, Some(&surface_props));
    let mut surface = surface_holder.unwrap();
    let canvas = surface.canvas();
    let mut paint = Paint::default();
    paint.set_style(Style::Fill);
    let rect = Rect::new(0.0, 0.0, 100.0, 100.0);
    canvas.draw_rect(rect, &paint);
}
```

Xcode output 
```objc
Undefined symbols for architecture x86_64:
  "SkFontParameters::Variation::Axis::setHidden(bool)", referenced from:
      skia_bindings::bindings::SkFontParameters_Variation_Axis::setHidden::h3732f0e5568a7c90 in libcanvasnative.a(skia_safe-37d69c1bedbf10ce.skia_safe.91mkqjfv-cgu.1.rcgu.o)
  "typeinfo for SkShader", referenced from:
      _C_SkShader_Deserialize in libcanvasnative.a(bindings.o)
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

Any pointer(s) will be appreciated 😃

--- a/skia-bindings/Cargo.toml
+++ b/skia-bindings/Cargo.toml
@@ -8,7 +8,7 @@ homepage = "https://github.com/rust-skia/rust-skia"
 repository = "https://github.com/rust-skia/rust-skia"
 license = "MIT"
 
-version = "0.12.10"
+version = "0.12.11"
 authors = ["LongYinan <lynweklm@gmail.com>", "Armin Sander <armin@replicator.org>"]
 edition = "2018"
 build = "build.rs"

--- a/skia-bindings/src/bindings.cpp
+++ b/skia-bindings/src/bindings.cpp
@@ -1165,6 +1165,10 @@ extern "C" bool C_SkFontParameters_Variation_Axis_isHidden(const SkFontParameter
     return self->isHidden();
 }
 
+extern "C" void C_SkFontParameters_Variation_Axis_setHidden(SkFontParameters::Variation::Axis* self, bool hidden) {
+    self->setHidden(hidden);
+}
+
 //
 // SkVertices
 //
@@ -1572,7 +1576,11 @@ extern "C" SkShader* C_SkShaders_Lerp2(const SkShader* red, const SkShader* dst,
 }
 
 extern "C" SkShader* C_SkShader_Deserialize(const void* data, size_t length) {
-    return dynamic_cast<SkShader*>(SkShader::Deserialize(SkFlattenable::Type::kSkShaderBase_Type, data, length).release());
+    // note: dynamic_cast may lead to a linker error here on iOS x86_64
+    // https://github.com/rust-skia/rust-skia/issues/146
+    // "typeinfo for SkShader", referenced from:
+    //      _C_SkShader_Deserialize in libcanvasnative.a(bindings.o)
+    return (SkShader*)(SkShader::Deserialize(SkFlattenable::Type::kSkShaderBase_Type, data, length).release());
 }
 
 //

--- a/skia-safe/Cargo.toml
+++ b/skia-safe/Cargo.toml
@@ -7,7 +7,7 @@ homepage = "https://github.com/rust-skia/rust-skia"
 repository = "https://github.com/rust-skia/rust-skia"
 license = "MIT"
 
-version = "0.12.10"
+version = "0.12.11"
 authors = ["Armin Sander <armin@replicator.org>"]
 edition = "2018"
 
@@ -18,7 +18,7 @@ svg = ["skia-bindings/svg"]
 
 [dependencies]
 bitflags = "1.0.4"
-skia-bindings = { version = "0.12.10", path = "../skia-bindings" }
+skia-bindings = { version = "0.12.11", path = "../skia-bindings" }
 # TODO: 1.3 breaks offscreen_gl_context
 lazy_static = "=1.2"
 

--- a/skia-safe/src/core/font_parameters.rs
+++ b/skia-safe/src/core/font_parameters.rs
@@ -4,7 +4,8 @@ pub mod variation {
     use crate::prelude::*;
     use crate::FourByteTag;
     use skia_bindings::{
-        C_SkFontParameters_Variation_Axis_isHidden, SkFontParameters_Variation_Axis,
+        C_SkFontParameters_Variation_Axis_isHidden, C_SkFontParameters_Variation_Axis_setHidden,
+        SkFontParameters_Variation_Axis,
     };
 
     #[derive(Clone, PartialEq, Default, Debug)]
@@ -37,9 +38,16 @@ pub mod variation {
         }
 
         pub fn set_hidden(&mut self, hidden: bool) -> &mut Self {
+            // does not link on iOS / x86_64:
+            // https://github.com/rust-skia/rust-skia/issues/146
+            /*
             unsafe {
                 self.native_mut().setHidden(hidden);
             };
+            */
+            unsafe {
+                C_SkFontParameters_Variation_Axis_setHidden(self.native_mut(), hidden);
+            }
             self
         }
     }

